Indicators/IndicatorBase.cs

```diff
@@ -77,7 +77,7 @@ protected IndicatorBase(string name)
         public bool Update(IBaseData input)
         {
             T _previousSymbolInput = default(T);
-            if (_previousInput.TryGetValue(input.Symbol.ID, out _previousSymbolInput) && input.EndTime < _previousSymbolInput.EndTime)
+            if (_previousInput.TryGetValue(input.Symbol.ID, out _previousSymbolInput) && input.EndTime <= _previousSymbolInput.EndTime)
```

Comment:
Interesting change! We should probably also add some new test covering this new behavior 👍

Reply:
I added this test. Fails on master, passes in this PR:
https://github.com/QuantConnect/Lean/pull/5442/files#diff-027dc7219a991f6c6abddceb033c0e0a2e841c166553845e0766b0d04403c18bR96